plutus-ledger-api/src/PlutusLedgerApi/V1.hs

```diff
@@ -3,99 +3,101 @@
 -- | The interface to Plutus V1 for the ledger.
 module PlutusLedgerApi.V1 (
   -- * Scripts
-  SerialisedScript,
-  ScriptForEvaluation,
-  serialisedScript,
-  deserialisedScript,
-  serialiseCompiledCode,
-  serialiseUPLC,
+  Common.SerialisedScript,
+  Common.ScriptForEvaluation,
+  Common.serialisedScript,
+  Common.deserialisedScript,
+  Common.serialiseCompiledCode,
+  Common.serialiseUPLC,
   deserialiseScript,
-  uncheckedDeserialiseUPLC,
+  Common.uncheckedDeserialiseUPLC,
 
   -- * Running scripts
   evaluateScriptRestricting,
   evaluateScriptCounting,
 
   -- ** Protocol version
-  MajorProtocolVersion (..),
+  Common.MajorProtocolVersion (..),
 
   -- ** Verbose mode and log output
-  VerboseMode (..),
-  LogOutput,
+  Common.VerboseMode (..),
+  Common.LogOutput,
 
   -- * Costing-related types
-  ExBudget (..),
-  ExCPU (..),
-  ExMemory (..),
-  SatInt (unSatInt),
-  fromSatInt,
+  Common.ExBudget (..),
+  Common.ExCPU (..),
+  Common.ExMemory (..),
+  Common.SatInt (unSatInt),
+  Common.fromSatInt,
 
   -- ** Cost model
-  EvaluationContext,
-  mkEvaluationContext,
-  ParamName (..),
-  CostModelApplyError (..),
-  CostModelParams,
-  assertWellFormedCostModelParams,
+  EvaluationContext.EvaluationContext,
+  EvaluationContext.mkEvaluationContext,
+  ParamName.ParamName (..),
+  EvaluationContext.CostModelApplyError (..),
+  EvaluationContext.CostModelParams,
+  EvaluationContext.assertWellFormedCostModelParams,
 
   -- * Context types
-  ScriptContext (..),
-  ScriptPurpose (..),
+  Contexts.ScriptContext (..),
+  Contexts.ScriptPurpose (..),
 
   -- ** Supporting types used in the context types
 
-  -- *** ByteStrings
-  BuiltinByteString,
-  toBuiltin,
-  fromBuiltin,
+  -- *** Builtins
+  Common.BuiltinByteString,
+  Common.toBuiltin,
+  Common.fromBuiltin,
+  Common.toOpaque,
+  Common.fromOpaque,
```

Comment:
I've also reexported `toOpaque` and `fromOpaque` from here, since `toBuiltin` and `fromBuiltin` used to subsume those.